controllers/machinedeletionremediation_controller.go

```diff
@@ -369,6 +399,35 @@ func (r *MachineDeletionRemediationReconciler) updateConditions(reason processin
 	return true, nil
 }
 
+func (r *MachineDeletionRemediationReconciler) setPermanentNodeDeletionExpectedCondition(status metav1.ConditionStatus, mdr *v1alpha1.MachineDeletionRemediation) bool {
+	var reason, message string
+	switch status {
+	case metav1.ConditionTrue:
+		reason = "brandNewNodeExpected"
```

Comment:
> not sure if "brand new" qualifies as technical term? At least in German it's more like marketing speech ;)

ah, it might be :)

> What about MachineDeletionOnCloudProviderCausesNewNodeName

seems good, also the BareMetal version :+1: